dev/TabView/TestUI/TabViewPage.xaml

```diff
@@ -24,6 +24,7 @@
                 <CheckBox x:Name="HandleTabItemCloseRequestedCheckBox" AutomationProperties.Name="HandleTabItemCloseRequestedCheckBox" Content="Handle TabViewItem close"/>
                 <CheckBox x:Name="IsAddButtonVisibleCheckBox" AutomationProperties.Name="IsAddButtonVisibleCheckBox" Content="Add button visible" IsChecked="{x:Bind Tabs.IsAddTabButtonVisible, Mode=TwoWay}"/>
                 <CheckBox x:Name="IsDisabledTabVisibleCheckBox" AutomationProperties.Name="IsDisabledTabVisibleCheckBox" Checked="IsDisabledTabVisibleCheckBox_CheckChanged" Unchecked="IsDisabledTabVisibleCheckBox_CheckChanged" Content="Disabled tab visible" IsChecked="True"/>
+                <CheckBox x:Name="IsTabViewEnabled" AutomationProperties.Name="IsTabViewEnabledCheckBox" Content="IsEnabled on TabView" IsChecked="{x:Bind Tabs.IsEnabled, Mode=TwoWay}"/>
```

Comment:
Any way we could turn this into a proper test?

Reply:
I think this is the last questions I have for this PR.

Reply:
> Any way we could turn this into a proper test?
> I think this is the last questions I have for this PR.
As I said, the best unit test for such kinds of errors would be to ensure that there are no visual state groups touching the same properties. I managed to implement such unit test, but want to discuss it in separate PR. Preview is available  here https://github.com/eugenegff/microsoft-ui-xaml/commit/db1908f0bb6df0fdc948875927f16aa6290522e0

Reply:
While I agree something like this would be valuable I think it would also be valueable to make a regression test that ensures that a renabled tabViewItem gets the correct visualization. This is probably pretty hard to do  though. what do you think?

Reply:
I don't know. I invested my time to create comprehensive test for the underlying issue of the conflicting visual groups. That test catches current issue too, and could (should?) be refactored to be used with all styles in MUX. I really do not see any reason to spend time on anything less.

Reply:
Commit https://github.com/eugenegff/microsoft-ui-xaml/commit/db1908f0bb6df0fdc948875927f16aa6290522e0 is right on top of current PR and could be added to current PR as regression test.